### PR TITLE
StudentSections table on TeacherHomepage

### PR DESCRIPTION
**Pull request by Erin007**
Teachers enroll in other teachers' sections often as part of professional development.  This change adds the StudentSections table to the TeacherHomepage so teachers can better manage their own enrollment in sections. The new prop `joinedSections` is passed to the TeacherHomepage to facilitate this change. This also floats the New Section button to the right so that it's more aligned with the other actions buttons on the page.
classroom sections i ve joined

### CODE REVIEW

**Comment by Erin007** [comment]
@tanyaparker

**Comment by Erin007** [comment]
Note: line 58 in `StudentSections` now seems counterintuitive because the user _is_ a teacher, but they are acting as a student in that section.  I don't know that it needs to be changed, but something to notice if we're confused about it in the future.

**Comment by tanyaparker** [comment]
Do you mean the line is confusing in code or looks confusing to the user?

**Comment by Erin007** [comment]
The line is confusing in the code because the component is being used on the `TeacherHomepage` but the prop is `isTeacher={false}`.  The user sees what they're supposed to.

**Comment by breville** [comment]
Can we rename the field to `viewAsTeacher` or similar to make it more clear?

Also, are we concerned about the styling differences between the two tables, especially the padding inside the header row?  

The red buttons also seem to come too close to the edge of their cells.

**Comment by islemaster** [comment]
I'm DOTD today and there's a laundry list of things going wrong.  Please assign review to somebody else.

File: apps/src/templates/studioHomepages/TeacherSections.jsx [hunk 1]
```diff
@@ -24,6 +24,7 @@ class TeacherSections extends React.Component {
       <div id="classroom-sections">
         <ContentContainer
           heading={i18n.sectionsTitle()}
+          description="The following classroom sections are sections you've created"
```
Comment (epeach) [inline apps/src/templates/studioHomepages/TeacherSections.jsx]: Can we add internationalization here?

File: apps/src/templates/studioHomepages/StudentSections.jsx [hunk 2]
```diff
@@ -10,6 +10,7 @@ export default class StudentSections extends Component {
     initialSections: PropTypes.array.isRequired,
     isRtl: PropTypes.bool.isRequired,
     canLeave: PropTypes.bool.isRequired,
+    isTeacher: PropTypes.bool
```
Comment (epeach) [inline apps/src/templates/studioHomepages/StudentSections.jsx:14]: Agree with Brendan, rename or comment to make the boolean purpose clearer.

Comment (Erin007) [reply apps/src/templates/studioHomepages/StudentSections.jsx:14]: `isTeacher` is passed around to lots of components for different reasons so I included a comment explaining its use in this context.

**Comment by Erin007** [comment]
@breville I made the styles more consistent and updated the screenshot. What do you think?

File: apps/i18n/common/en_us.json [hunk 3]
```diff
@@ -597,6 +596,8 @@
   "sectionsNotificationJoinSuccess": "You joined section {sectionName}.",
   "sectionsNotificationLeaveSuccess": "You left section {sectionName}.",
   "sectionsNotificationSuccess": "Success!",
+  "sectionsJoinedDescription": "The following classroom sections are sections that you've joined",
```
Comment (breville) [inline apps/i18n/common/en_us.json]: I think we should have a `.` or `:` at the end of these sentences.

**Comment by breville** [comment]
Styling improvements look good though now I wonder whether a casual glance will make it clear enough how these are different to each other.  Should we have some kind of bolding of some keywords, or some additional header, or some colour changes to the table itself just to help differentiate?  @tanyaparker

**Comment by Erin007** [comment]
@breville we opted to separate the tables and give the Classroom Sections I've Joined text more prominence. I think this will help distinguish between them, while maintaining style consistency. What do you think?

**Comment by tanyaparker** [comment]
Can it be "Classroom Sections I've Joined" to have consistent capitalization? LGTM

**Comment by breville** [comment]
I think this is good.  It might be nice for the Classroom Sections heading to be in the row as the button on the right, to avoid that big gap before the able, but this isn't crucial.

**Comment by tanyaparker** [comment]
Woooo

**Pull request merged**